### DISCUSSION TITLE
500 server error after plugin installation

### DISCUSSION THREAD

**Discussion by klarahimmel67** [kimai/kimai#3086] [General]
hallo,
i installed kimai and it was working good.
https://www.kimai.org/documentation/installation.html
after that i bought the translation plugin and installed it like it is told here:
https://www.kimai.org/documentation/plugins.html#plugin-installation
now i get an error:

**Oops! An Error Occurred
The server returned a "500 Internal Server Error".
Something is broken. Please let us know what you were doing when this error occurred. We will fix it as soon as possible. Sorry for any inconvenience caused.**

can anybody help me?

**Comment by klarahimmel67** [comment]
the logfile says:

 Doctrine\\DBAL\\Driver\\PDO\\Exception(code: 42S02): SQLSTATE[42S02]: Base table or view not found: 1146 Table 'timetracker.kimai2_translations' doesn't exist at /var/www/html/suitecrm/timetracker/kimai2/vendor/doctrine/dbal/lib/Doctrine/DBAL/Driver/PDO/Exception.php:18, PDOException(code: 42S02): SQLSTATE[42S02]: Base table or view not found: 1146 Table 'timetracker.kimai2_translations' doesn't exist at /var/www/html/suitecrm/timetracker/kimai2/vendor/doctrine/dbal/lib/Doctrine/DBAL/Driver/PDOStatement.php:117)"} []

i copied the files to the appropriate folder ( /var/plugins/TranslationBundle ) and cleared the cache, like it is told here.
https://www.kimai.org/documentation/plugins.html#plugin-installation

the db table is missing ...

**Comment by klarahimmel67** [comment]
after deleting the plugin files and clearing the cache again i can log in again.
how can i install a plugin? i did this:

```
Make sure that you copy it into the appropriate subdirectory in var/plugins/. For the plugin CustomCSSBundle this would be var/plugins/CustomCSSBundle/.
After the plugin was installed you need to clear the cache:
How to reload Kimai cache
bin/console kimai:reload --env=prod
```

**Comment by kevinpapst** [comment]
Here you go: https://www.kimai.org/store/translation-bundle.html#installation

Scroll down to "Create database" which is what you are missing: `bin/console kimai:bundle:translation:install`

The bundled file README.md should contain that info as well. 

Let me know if it works now!

**Comment by klarahimmel67** [comment]
hallo, it is working now, thank you. great work! 

Am 21.01.2022 09:21, schrieb Kevin Papst:

> Here you go: https://www.kimai.org/store/translation-bundle.html#installation 
> 
> Scroll down to "Create database" which is what you are missing: bin/console kimai:bundle:translation:install 
> 
> The bundled file README.md should contain that info as well. 
> 
> Let me know if it works now! 
> 
> --
> Reply to this email directly, view it on GitHub [1], or unsubscribe [2].
> Triage notifications on the go with GitHub Mobile for iOS [3] or Android [4]. 
> You are receiving this because you authored the thread.Message ID: ***@***.***>

  

Links:
------
[1]
https://github.com/kevinpapst/kimai2/discussions/3086#discussioncomment-2012880
[2]
https://github.com/notifications/unsubscribe-auth/AXMZGYSC7LDMZL3CY3IO56TUXEJSBANCNFSM5MOVCHQQ
[3]
https://apps.apple.com/app/apple-store/id1477376905?ct=notification-email&amp;mt=8&amp;pt=524675
[4]
https://play.google.com/store/apps/details?id=com.github.android&amp;referrer=utm_campaign%3Dnotification-email%26utm_medium%3Demail%26utm_source%3Dgithub
